nix_update/__init__.py

```diff
@@ -26,6 +26,12 @@ def parse_args() -> Options:
     parser.add_argument(
         "--commit", action="store_true", help="Commit the updated package"
     )
+    parser.add_argument(
+        "-ve",
+        "--versionextractor",
```

Comment:
Ideally the name should already reflect that it is a regex i.e.: `--version-regex`